vector/v.surf.rst/test_suite/profile.sh

```diff
@@ -5,7 +5,7 @@ OUTFILE=callgrind.out.1196
 g.copy vect=elev_lid792_bepts,lidar_test
 v.build lidar_test
 g.region n=221230 s=220816 w=639090 e=639530 res=0.5 -p
-CMD="v.surf.rst in=lidar_test elev=omp_test.rst --o"
+CMD="v.surf.rst in=lidar_test elev=omp_test.rst nproc=8 --o"
```

Comment:
We might not be able to test this well, in CI, where we do not have 8 cores available.

Reply:
I removed the change in this file.

Reply:
If it is possible to use the amount of cores available, or at least 2, it might better test a non-serial invocation